src/utils/NewFilterTodos.tsx

```diff
@@ -0,0 +1,20 @@
+import { FilterBy } from './FilterBy';
+import { Todo } from '../types/Todo';
+
+export const NewFilterTodos = (
```

Comment:
```suggestion
export const getFilteredTodos = (
```
It's not a React component, use camelCase instead of PascalCase